perllib/FixMyStreet/DB/ResultSet/Problem.pm

```diff
@@ -159,7 +159,7 @@ sub _recent {
             $probs = [ $rs->search({
                 id => [ map { $_->id } @$probs ],
                 %$query,
-            })->all ];
+            }, $attrs)->all ];
```

Comment:
I don't see a reason why this search shouldn't include $attrs in the same way as the search below does.

Sorry if I'm missing something...